android/source/VideoLocker/src/org/edx/mobile/view/BaseTabActivity.java

```diff
@@ -62,6 +63,15 @@ private void setUpTabs() {
         }
     }
 
+    protected Fragment getFragmentByTag(String tag) {
+        if (fragmentManager != null) {
```

Comment:
How does the fragment manager end up null? Because we're on an old version of android? Maybe worth adding a comment.